contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -141,10 +141,12 @@ def _eval_subtree(repo_url, subtree):
     @staticmethod
     def _eval_repo_url(repo_url):
         """Allow passing short GitHub style URLs"""
+        has_git_extension = re.compile('.git$')
         if len(repo_url.split('/')) == 2 and not "git@" in repo_url:
-            return "https://github.com/{}.git".format(repo_url)
```

Comment:
I'm not that big of a fan of this implicit approach, but I know why it was done (to make chatops command more user-friendly).

Also, it looks like that right now we default to http (non ssh) URL if user doesn't provide a full url, is this correct?